src/defaultSettings.js

```diff
@@ -80,6 +80,19 @@ export function syncSettings(settings) {
     settings.marks[0].per = [settings.color_by];
     settings.margin = settings.margin || { bottom: settings.time_settings.vertical_space };
 
+    //Convert unscheduled_visit_pattern from string to regular expression.
+    if (
+        typeof settings.unscheduled_visit_pattern === 'string' &&
+        settings.unscheduled_visit_pattern !== ''
+    ) {
+        const flags = settings.unscheduled_visit_pattern.replace(/.*?\/([gimy]*)$/, '$1'),
+            pattern = settings.unscheduled_visit_pattern.replace(
+                new RegExp('^/(.*?)/' + flags + '$'),
+                '$1'
+            );
+        settings.unscheduled_visit_regex = new RegExp(pattern, flags);
+    }
+
```

Comment:
This looks good - should also make it easier to support `config.unscheduled_visit_list` as suggested in #47 code review